index.html

```diff
@@ -59,25 +61,53 @@
           alert("Browser support not implemented yet")
         }
       }
+
+      function closeMenu(){
+        let drawer = document.getElementById('menu');
+        drawer.classList.remove('is-visible');
+        let obfuscator = document.getElementsByClassName('mdl-layout__obfuscator')[0];
+        obfuscator.classList.remove('is-visible');
+      }
+
+      function closeMenuViews(){
+        let views = document.getElementsByClassName("user-page");
+        for (var i = 0; i < views.length; i++)
+          views[i].style.display = 'none';
+      }
+
+      function showProfile() {
+        closeMenuViews();
+        let view = document.getElementById('profile-view');
+        view.style.display = 'block';
+        closeMenu();
+      }
+
+      function showGame(event){
+        closeMenuViews();
+        let view = document.getElementById('game-view');
+        view.style.display = 'block';
+        view.setAttribute('name', event.name)
+        closeMenu();
+      }
     </script>
 
   </head>
   <body>
     <mdl-layout>
 
-      <mdl-drawer>
+      <mdl-drawer id="menu">
         <span class="mdl-layout-title">Menu</span>
         <mdl-navigation>
-          <a>
+          <a onclick="showProfile()" style="cursor: pointer;">
```

Comment:
Also, making classes gives out code which is more clear (style is separated from the rest).